lizmap/modules/lizmap/classes/lizmap.class.php

```diff
@@ -322,28 +322,11 @@ public static function getProject($key)
      */
     public static function getLogConfig()
     {
-        return jClasses::getService('lizmap~lizmapLogConfig');
-    }
-
-    /**
-     * Get a list of log items names.
-     *
-     * @return string[] list of names
-     */
-    public static function getLogItemList()
```

Comment:
We should not remove this method, to avoid issues with existing external modules, and to avoid an issue in `getLogItem()`.
Mark it as deprecated, and change its code to use new methods